.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "DeepTiling"]
```

Comment:
Unfortunately I need this project to be installable as a package by pip, and pip can't handle git submodules. Therefore I think the best way to handle the (unmodified) parent class is to include the parent class in this repository as you did before. Because it is unmodified it will still be reasonably easy to update it if that is ever nevessary,

Reply:
okay, i will change it folder instead of submodule.